src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -280,6 +280,8 @@ private void ChooseActions(in Entity entity, ref MicrobeAI ai, ref CompoundAbsor
 
         ref var control = ref entity.Get<MicrobeControl>();
 
+        ref var cellHealth = ref entity.Get<Health>();
```

Comment:
Wouldn't it be better to use the `health` variable that is already passed into this function?